lib/src/utils/json_helper.dart

```diff
@@ -169,6 +170,7 @@ class JsonHelper {
               height: JsonObject.parseInt(numberObject[_ALL_IMAGES_TAG_HEIGHT]),
               url: numberObject[_ALL_IMAGES_TAG_URL],
               uploaded: uploaded,
+              contributor: fieldObject[_ALL_IMAGES_TAG_UPLOADER],
```

Comment:
`_ALL_IMAGES_TAG_UPLOADED` appears in both `allImagesToJson` and `allImagesFromJson`.
In your code `_ALL_IMAGES_TAG_UPLOADER` appears only once. Probably something needs to be fixed.
Additional unit tests would help, too.

Reply:
I didn't include the 'to' method on purpose, because today, images are well assigned to the contributor.
But I can add do, no pb.

In terms of tests, despite checking the result of the server, I don't really see any benefits here.